src/website/app/pages/index.js

```diff
@@ -50,6 +51,11 @@ const pages: Pages = [
     component: Theming,
     path: '/theming',
     title: 'Theming'
+  },
+  {
+    component: ComponentStatus,
```

Comment:
What do you think about linking to this page from the ‘Components’ heading, rather than create a new ‘Component Status’ navigation item?

Reply:
I added this link to be the heading -> **Component** [status]